src/main/java/uk/gov/di/services/ValidationService.java

```diff
@@ -20,4 +22,24 @@
         }
         return emailErrors;
     }
+
+    public Set<PasswordValidation> validatePassword(String password, String retypedPassword) {
```

Comment:
In principle, all looks good. We could code a bit more defensively here - if either of the inputs are `null` then this method will NPE.

In practice, this _shouldn't_ happen if we're passing them through from the Dropwizard inputs (I think it turns them into empty strings) but an extra clause to cover our bases, perhaps?

Reply:
Yep you're right, it does convert them to an empty String. But not a bad shout, I'll add something in.

Reply:
I've updated and added a test in for good measure